vendor.js

```diff
@@ -0,0 +1,34 @@
+//vendor module
+//dependencies
+const faker = require('faker');
+const events = require('./events');
+const name = require(process.env.STORE_NAME);
+//use faker
+app.use(faker);
+//faker objects
+const randomName = (faker.name.lastName(), faker.name.firstName());
+const randomOrderId = faker.random.number();
+const randomCity = (faker.address.city(), faker.address.state(), faker.address.zipCode()); 
+//sets a 5 sec time lapse to send an event notification
+setInterval(()=> {
+//object using faker vars to pass a fake order
+    let payload = {
+        storeName: name,
+        orderId: randomOrderId,
```

Comment:
When I run your application, it gives me an error with id saying that it needs to be a string but is receiving undefined